### ISSUE TITLE
Keyboard navigation plugin unregister/re-register copy/cut/paste shortcuts

### ISSUE THREAD

**Issue by changminbark** [rachel-fenichel/blockly-samples#2424]
### Check for duplicates

- [X] I have searched for similar issues before opening a new one.

### Component

Keyboard Navigation Plugin

### Problem

Currently, when combining the keyboard navigation plugin with the multiselect plugin, the copy/cut/paste shortcuts overlap. That is why the multiselect mode unregisters its copy/cut/paste shortcuts when the keyboard navigation mode is on and re-registers it when the keyboard navigation  mode is turned off. However, this leads to an issue where the user cannot use the original copy/cut/paste shortcuts with the cursor while the keyboard navigation mode is on.

### Request

One of the solutions to this would be to introduce a way to unregister/re-register the keyboard navigation plugin's copy/cut/paste shortcuts. That way, we can unregister/re-register the original Blockly core's copy/cut/paste shortcuts without running into a collision error. 

More information regarding the issue from the multiselect side: https://github.com/mit-cml/workspace-multiselect/issues/50#issuecomment-2250364202

![image](https://github.com/user-attachments/assets/f024e224-1a39-44ce-bf03-f5bc15687fd3)

### Alternatives considered

The alternative solution to this would be to allow the original Blockly core's copy/cut/paste shortcuts to allow for collisions. However, this solution seems less elegant.

**Comment by BeksOmega** [comment]
Heya @changminbark Thank you for reporting =) This is something we'll keep in mind as we test and refactor keyboard navigation! But I wouldn't consider compatibility with keyboard nav a blocker for your multiselect work. Keyboard nav is still a beta plugin, that we don't think is well suited to users yet.

**Comment by gonfunko** [comment]
Closing since the keyboard navigation plugin has been removed.
